generators/entity-server/files.js

```diff
@@ -34,6 +34,40 @@ const SERVER_TEST_SRC_DIR = constants.SERVER_TEST_SRC_DIR;
 // In order to have consistent results with Faker, the seed is fixed.
 faker.seed(42);
 
+/*
+ * Current faker version is 4.1.0 and was release in 2017
+ * It is outdated
+ * https://github.com/Marak/faker.js/blob/10bfb9f467b0ac2b8912ffc15690b50ef3244f09/lib/date.js#L73-L96
+ * Needed for reproducible builds
+ */
+/* eslint-disable-next-line */
+const recentDate = function (days, refDate) {
```

Comment:
if this is a function, please a "verbal" form (getRecentDate for instance)
What does this function do and why? I don't get it just by looking at the name

Reply:
This is an exact update faker recent function with refDate parameter.
Faker lib seem unsupported right now, with no recent release.
Wanted to keep it the same.
https://github.com/Marak/faker.js/blob/9fd8d7d37b398842d0784a116a340f7aa6afb89b/lib/date.js#L73-L96

PR
https://github.com/Marak/faker.js/commit/816b658e26badba6a911b7065aa8ca413d848b9d#diff-9e4794cb4c334b7eaa3dfdff8725406e

Reply:
Ah okay, thanks for the explanation.
My point still applies though: if we introduce some code in our system, it has to abide to the same level of quality (or above) as the rest of the code.